packages/components/src/RecurringSelect/components/DayOfMonthSelect/DayOfMonthSelect.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { v1 as uuidv1 } from "uuid";
+import styles from "./DayOfMonthSelect.css";
+import checkboxStyles from "../../DateCellCheckbox.css";
+import { DayOfMonth } from "../../types";
+
+interface DayOfMonthSelectProps {
+  selectedDays: Set<DayOfMonth>;
+  disabled: boolean;
+  onChange(selectedDays: Set<DayOfMonth>): void;
+}
+
+const daysInMonth = [...Array.from({ length: 31 }, (_, i) => i + 1), "LAST"];
+
+export function DayOfMonthSelect({
+  selectedDays,
+  disabled,
+  onChange,
+}: DayOfMonthSelectProps) {
+  const onChangeDayOfMonth = (
```

Comment:
just a best practice thing in react and js land. it's kinda on most of the components here as well

`on` is what you'd use as a prop, `handle` is what you feed in to said prop. I.e., `onClick={handleClick}` 

also would prefer if it's a function below the return and not an arrow one. No biggie tho.